kore/src/Kore/Builtin/String.hs

```diff
@@ -297,26 +298,25 @@ evalFind = Builtin.functionEvaluator evalFind0
     evalFind0 _ _ _ = Builtin.wrongArity findKey
 
 evalString2Base :: BuiltinAndAxiomSimplifier
-evalString2Base = Builtin.functionEvaluator evalString2Base0
+evalString2Base = Builtin.functionEvaluatorWithApp evalString2Base0
   where
-    evalString2Base0 _ resultSort [_str, _base] = do
-        _str <- expectBuiltinString string2BaseKey _str
-        _base <- Int.expectBuiltinInt string2BaseKey _base
-        let readN =
-                case _base of
-                    -- no builtin reader for number in octal notation
-                    8 -> \s ->
-                        case readOct $ Text.unpack s of
-                            [(result, "")] -> Right (result, "")
-                            _ -> Left ""
-                    10 -> Text.signed Text.decimal
-                    16 -> Text.signed Text.hexadecimal
-                    _ -> const empty
```

Comment:
Just so I'm 100% sure I understand, what caused this function to return `\\bottom` before, instead of just not evaluating?

Reply:
Got it, thanks!